src/runtime/virtcontainers/kata_agent.go

```diff
@@ -98,6 +98,7 @@ var (
 	kata9pDevType                 = "9p"
 	kataMmioBlkDevType            = "mmioblk"
 	kataBlkDevType                = "blk"
+	kataDmVerityBlkDevType        = "dmverity"
```

Comment:
Switch line 101 and 102 so we group all block device together?

Reply:
Done.